contribution/mobile/ja/procedural-names.json

```diff
@@ -207,40 +199,43 @@
 		"震える",
 		"トリニティーの",
 		"夕暮れの",
-		"夕暮れの",
 		"ねじれる",
 		"台風の",
 		"ヴァルキリーの",
 		"覇者の",
 		"復讐の",
 		"ベテランの",
-		"ベテランの",
 		"勝者の",
 		"奈落に落ちる",
 		"渦動の",
 		"ウォーフォージドの",
 		"ウォーブリンガーの",
 		"看守の",
 		"軍閥の",
-		"ウォーモンジャーの",
-		"ウォーモンジャーの",
-		"ワープする",
+		"武器商人の",
 		"ワープする",
 		"戦士の",
 		"戦士の",
 		"囁きの",
 		"笛吹く"
 	],
-	"EQUIPMENT_TYPES_INSTRUCTIONS": "アクティベートするには最低10個のパーツが必要です",
+	"EQUIPMENT_TYPES_INSTRUCTIONS": "ゲーム内で使用されるためには最低10個のパーツが必要です",
 	"EQUIPMENT_TYPES": {
 		"PRIMARY_WEAPON": {
 			"names": [
+				"弓",
+				"カービン銃",
+				"クロスボウ",
+				"銃",
+				"ライトセイバー",
 				"拳銃",
 				"光線銃",
 				"ブラスター",
 				"銃",
 				"シューター",
-				"ザッパー"
+				"ザッパー",
+				"水鉄砲",
+				"輪ゴム鉄砲",
```

Comment:
Please erase the comma (line 238) as it's the last variable in the array. Adding a comma without any final variable will result in an error.